src/FullNetworkSystems.jl

```diff
@@ -10,7 +10,18 @@ export System, SystemDA, SystemRT
 export Zone, Generator, Bus, Branch
 export GeneratorTimeSeries, GeneratorStatus, GeneratorStatusDA, GeneratorStatusRT
 export gens_per_zone, branches_by_breakpoints, get_datetimes
+export get_static_components, get_zones, get_buses, get_generators, get_branches
+export get_regulation_requirements, get_operating_reserve_requirements
+export get_gens_per_bus, get_loads_per_bus, get_incs_per_bus, get_decs_per_bus, get_psds_per_bus
+export get_ptdf, get_lodf
+export get_initial_commitment, get_bids, get_availability, get_must_run
+export get_initial_generation, get_load, get_offer_curve
+export get_pmin, get_pmax, get_regmin, get_regmax
+export get_regulation, get_spinning, get_supplemental_on, get_supplemental_off
+export get_commitment_status, get_commitment_reg_status
```

Comment:
I think we can use this opportunity to make these names shorter. "commitment status" is a bit redundant, we could have these as

```suggestion
export get_commitment, get_regulation_commitment
```

which I think makes the regulation one clear as well